tests/components/wemo/conftest.py

```diff
@@ -0,0 +1,81 @@
+"""Fixtures for pywemo."""
+import asynctest
+import pytest
+import pywemo
+
+from homeassistant.components.wemo import CONF_DISCOVERY, CONF_STATIC
+from homeassistant.components.wemo.const import DOMAIN
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+
+MOCK_HOST = "127.0.0.1"
+MOCK_PORT = 50000
+MOCK_NAME = "WemoDeviceName"
+MOCK_SERIAL_NUMBER = "WemoSerialNumber"
+
+
+@pytest.fixture(name="pywemo_model")
+def pywemo_model_fixture():
+    """Fixture containing a pywemo class name used by pywemo_device_fixture."""
+    return "Insight"
+
+
+@pytest.fixture(name="pywemo_registry")
+def pywemo_registry_fixture():
+    """Fixture for SubscriptionRegistry instances."""
+    registry = asynctest.create_autospec(pywemo.SubscriptionRegistry)
+
+    registry.callbacks = {}
+
+    def on_func(device, type_filter, callback):
+        registry.callbacks[device.name] = callback
+
+    registry.on.side_effect = on_func
+
+    with patch("pywemo.SubscriptionRegistry", return_value=registry):
+        yield registry
+
+
+@pytest.fixture(name="pywemo_device")
+def pywemo_device_fixture(pywemo_registry, pywemo_model):
+    """Fixture for WeMoDevice instances."""
+    device = asynctest.create_autospec(getattr(pywemo, pywemo_model))
+    device.host = MOCK_HOST
+    device.port = MOCK_PORT
+    device.name = MOCK_NAME
+    device.serialnumber = MOCK_SERIAL_NUMBER
+    device.model_name = pywemo_model
+
+    with patch(
+        "homeassistant.components.wemo.validate_static_config", return_value=device
+    ):
+        yield device
+
+
+@pytest.fixture
+def event_loop(hass):
```

Comment:
This fixture shouldn't be needed.

Reply:
Thanks. I removed the fixture as you suggested. Testing via tox works just fine without overriding the default `async_loop` fixture.

Looks like an issue on my end. I must have something broken in my venv. My workflow had be testing via pytest from my venv, which was failing without overriding the default `async_loop` fixture:

```
RuntimeError: Task <Task pending name='Task-82' coro=<pytest_fixture_setup.<locals>.wrapper.<locals>.setup() running at venv/lib/python3.8/site-packages/pytest_asyncio/plugin.py:110> cb=[_run_until_complete_cb() at /usr/lib/python3.8/asyncio/base_events.py:180]> got Future <Task pending name='Task-83' coro=<_async_setup_component() running at homeassistant/setup.py:128>> attached to a different loop
```

Something for me to troubleshoot later.